apps/osie-installer.sh

```diff
@@ -135,9 +133,15 @@ cat <<-EOF | grep -v '^\s*$'
 	${packet_base_url:+packet_base_url: $packet_base_url}
 EOF
 
-until docker info; do
+service docker start
+i=0
+# shellcheck disable=SC2169
+until docker info >/dev/null || [[ $i -gt 5 ]]; do
```

Comment:
see below

Reply:
doh, I figured it put these comments in order of the code, not in the order that I left them...  anyway.. you get the idea ;)

Reply:
heh